### ISSUE TITLE
Add filter to set contentSelector

### ISSUE THREAD

**Issue by soderlind** [westonruter/spoken-word#1]
I would be nice if the following was wrapped in an filter https://github.com/westonruter/spoken-word/blob/e0174587806802ba3668ce43e57f0429022d1d08/spoken-word.php#L88

eg:

```php
'contentSelector' =>  apply_filters( 'spoken_word_content_selector',  '.hentry .entry-content, .h-entry .e-content, [itemprop="articleBody"]' ),
```

**Comment by westonruter** [comment]
Good call. I think actually it should be a `spoken_word_options` filter that wraps all of the `$export` array:

https://github.com/westonruter/spoken-word/blob/e0174587806802ba3668ce43e57f0429022d1d08/spoken-word.php#L87-L90

**Comment by westonruter** [comment]
See also #3 which could eliminate the need to customize the `contentSelector`.

**Comment by eri-trabiccolo** [comment]
Hi there,
FWIW I do think the `spoken_word_options` filter would be really useful, especially for theme compatibility. 
E.g. some themes might remove the `hentry` class from the `post_class()` array, mostly to avoid google search console structured data errors on pages (_author_ and _updated_ properties are typically missing in pages, but required for the `hentry` microformat)

#3 would be great too, but it can coexist with this filter, making the `$export` something like the following:
```php
 	$exports = array( 
 		'contentSelector' => '.hentry .entry-content, .h-entry .e-content, [itemprop="articleBody"]', 
 		'useDashicons'    => true,
                'autodetectContent' => false,
 	); 
```
when `autodetectContent` is `true` the `contentSelector` option will not be used.

Best
